src/main/java/net/neoforged/neoforge/common/extensions/IBlockEntityExtension.java

```diff
@@ -103,13 +104,15 @@ default AABB getRenderBoundingBox() {
         BlockState state = self().getBlockState();
         Block block = state.getBlock();
         BlockPos pos = self().getBlockPos();
-        if (block == Blocks.ENCHANTING_TABLE) {
-            bb = new AABB(pos, pos.offset(1, 1, 1));
-        } else if (block == Blocks.CHEST || block == Blocks.TRAPPED_CHEST) {
+        if (block == Blocks.DRAGON_HEAD) {
+            bb = new AABB(pos.getX() - .5, pos.getY(), pos.getZ() - .5, pos.getX() + 1.5, pos.getY() + 1.5, pos.getZ() + 1.5);
+        } else if (block == Blocks.BELL || block == Blocks.CAMPFIRE || block == Blocks.SOUL_CAMPFIRE || block instanceof SkullBlock) {
+            bb = new AABB(pos);
+        } else if (block == Blocks.CHEST || block == Blocks.TRAPPED_CHEST || block == Blocks.ENDER_CHEST) {
             bb = new AABB(pos.offset(-1, 0, -1), pos.offset(2, 2, 2));
-        } else if (block == Blocks.STRUCTURE_BLOCK) {
```

Comment:
This seems a bit weird to me....? Is it GitHub that is stupid or did you remove the infinite bounds on the structure block?

Reply:
The initial value of `bb` is the infinite extent AABB, and the last condition in this `if`-`else` chain now checks that the block is _not_ the structure block. Just a small cleanup to reduce a bit of unnecessary duplication.